android/build.gradle

```diff
@@ -1,17 +1,17 @@
 buildscript {
-    ext.kotlin_version = '1.9.25'
+       ext.kotlin_version = '1.9.25'
     ext {
        compileSdkVersion   = 34                // or higher / as desired
        targetSdkVersion    = 34                // or higher / as desired
    }
-
     repositories {
         google()
         mavenCentral()
     }
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.android.tools.build:gradle:7.3.1'
```

Comment:
This is min version. we use gradle 8.3 (from gradle-wrapper.properties) so this is not necessary